src/biokbase/narrative/jobs/appmanager.py

```diff
@@ -435,26 +436,43 @@ def run_app_bulk(
 
         Example:
         --------
-        run_app_batch([{
-            "app_id": "Some_module/reads_to_contigset",
-            "tag": "release",
-            "version": "1.0.0",
-            "params": [{
-                "read_library_name" : "My_PE_Library",
-                "output_contigset_name" : "My_Contig_Assembly"
-            }, {
-                "read_library_name": "Another_reads_library",
-                "output_contigset_name": "Another_contig_assembly"
-            }]
-        }, {
-            "app_id": "Some_module/contigset_to_genome",
-            "tag": "release",
-            "version": "1.1.0",
-            "params": [{
-                "contigset": "My_contigset",
-                "genome_name": "My_genome"
-            }]
-        }])
+        run_app_bulk(
```

Comment:
Mostly some reformatting (so I could understand it) and pulling out the `shared_params`